api-reference/50 Common/Object Structures/positionConfig/of.md

```diff
@@ -4,32 +4,324 @@ type: String | Node | jQuery | Window
+
+    ---
+
+- <a href="https://developer.mozilla.org/en-US/docs/Web/API/Window" target="_blank">Window</a>
+
+    ---
+    ##### jQuery
+    
+        <!-- tab: index.js -->
+        $(function() {
+            $("#popupContainer").dxPopup({
+                // ...
+                position: {
+                    of: window
+                }
+            });
+        });
+    
+    ##### Angular
+
+        <!-- tab: app.component.html -->
+        <dx-popup>
+            <dxo-position
+                [of]="getWindow()">
+            </dxo-position>
+        </dx-popup>
+    
+        <!-- tab: app.component.ts -->
+        import { Component } from '@angular/core';
+    
+        @Component({
+            selector: 'app-root',
+            templateUrl: './app.component.html',
+            styleUrls: ['./app.component.css']
+        })
+        export class AppComponent {
+            getWindow() {
+                return window;
+            }
+        }
+    
+    ##### Vue
+    
+        <!-- tab: App.vue -->
+        <template>
+            <DxPopup>
+                <DxPosition :of="target" />
+            </DxPopup>
+        </template>
+    
+        <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+    
+        import DxPopup, {
+            DxPosition
+        } from 'devextreme-vue/popup';
+    
+        export default {
+            components: {
+                DxPopup,
+                DxPosition
+            },
+            data() {
+                return {
+                    target: window
+                }
+            }
+        }
+        </script>
+    
+    ##### React
+    
+        <!-- tab: App.js -->
+        import React from 'react';
+    
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+    
+        import Popup, {
+            Position
+        } from 'devextreme-react/popup';
+    
+        class App extends React.Component {
+            constructor(props) {
+                super(props);
+                this.target = window;
+            }
+            render() {
+                return (
+                    <Popup>
+                        <Position
+                            of={this.target}
+                        />
+                    </Popup>
+                );
+            }
+        }
+        export default App;
+    
+    ---
+
+- [dxEvent](/api-reference/50%20Common/Object%20Structures/dxEvent '/Documentation/ApiReference/Common/Object_Structures/dxEvent/') (or <a href="http://api.jquery.com/category/events/event-object/" target="_blank">jQuery.Event</a> if jQuery is used)
+
+ The overlay element is positioned at the **event.pageX** and **event.pageY** coordinates. In the following example, the [Popover](/api-reference/10%20UI%20Widgets/dxPopover '/Documentation/ApiReference/UI_Widgets/dxPopover/') widget is positioned at a clicked point on the target.
```

Comment:
In the second sentence, do you mean:
In the following example, the [Popover](/api-reference/10%20UI%20Widgets/dxPopover '/Documentation/ApiReference/UI_Widgets/dxPopover/') widget is positioned at the point where the user has clicked.